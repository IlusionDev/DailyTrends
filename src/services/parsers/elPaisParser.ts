import { JSDOM } from "jsdom";

export default function (pageContent: string) {
  const dom = new JSDOM(pageContent);
  const newElements = dom.window.document.querySelectorAll("h2");
  const news = [];
  newElements.forEach((element) => {
    const aElement = element.getElementsByTagName("a").item(0);
    const title = aElement?.text;
    const url = aElement?.attributes.getNamedItem("href")?.value;
    const image = element.parentElement.getElementsByTagName("img");
    let images = {
      default: image.item(0)?.attributes.getNamedItem("src")?.value,
    };

    news.push({
      url,
      title,
      images,
      site: "elPais",
    });
  });

  return news;
}
