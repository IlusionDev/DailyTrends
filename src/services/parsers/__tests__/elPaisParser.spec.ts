import { JSDOM } from 'jsdom'
import fs from 'fs'
import path from 'path'

const HTML_FILE = path.resolve(__dirname, "../__mocks__", "elPaisWebPageMock.html");
let elPaisPageMock = fs.readFileSync(HTML_FILE, 'utf8')
let dom = new JSDOM(elPaisPageMock);
let getElementsByClassName
let getElementsByTagName
let getElementsByTagNameFromParent
const documentMock = {
       window: {
        document: {
           querySelectorAll: jest.fn((param) => {
             const elements = dom.window.document.querySelectorAll(param)
             const element = elements.item(0)
             getElementsByClassName = jest.spyOn(element, "getElementsByClassName")
             getElementsByTagName = jest.spyOn(element, "getElementsByTagName")
             getElementsByTagNameFromParent = jest.spyOn(element.parentElement, "getElementsByTagName")

             return elements
           }),
         },
         ...dom
       }
    }
const jsdommock = {
    JSDOM: jest.fn(() => documentMock)
}

jest.doMock('jsdom', () => jsdommock)
import elPaisParser from "@/services/parsers/elPaisParser";

describe("El mundo parser", () => {
  it("creates a error api", () => {
    
    const models = elPaisParser(elPaisPageMock)

    expect(documentMock.window.document.querySelectorAll).toHaveBeenNthCalledWith(1, "h2")
    expect(getElementsByTagName).toHaveBeenNthCalledWith(1, "a")
    expect(getElementsByTagNameFromParent).toHaveBeenNthCalledWith(1, "img")
    expect(models).toMatchSnapshot()
  })
})