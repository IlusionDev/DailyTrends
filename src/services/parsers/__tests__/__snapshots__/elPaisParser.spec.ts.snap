// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`El mundo parser creates a error api 1`] = `
Array [
  Object {
    "images": Object {
      "default": "https://imagenes.elpais.com/resizer/Fx54Apjlenj3THvo6lbyDKmtxFY=/62x34/filters:focal(3368x1138:3378x1148)/cloudfront-eu-central-1.images.arcpublishing.com/prisa/JVZ6VISIH5HLZE2YHZCS6WBRJI.jpg",
    },
    "site": "elPais",
    "title": "Bruselas lanza la Autoridad Europea para combatir el blanqueo ",
    "url": "/economia/2021-07-19/bruselas-propone-una-autoridad-europea-para-combatir-el-blanqueo-de-capitales.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Los turistas británicos vuelven a España: las reservas de vuelos del Reino Unido se disparan un 400%",
    "url": "/economia/2021-07-19/vuelven-los-turistas-britanicos-las-reservas-de-vuelos-del-reino-unido-se-disparan-un-400.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La falsa normalidad de Xinjiang
",
    "url": "/internacional/2021-07-19/el-xinjiang-de-xi-jinping.html",
  },
  Object {
    "images": Object {
      "default": "https://imagenes.elpais.com/resizer/9BaFHBtzBjsy3cqhzhBRYqEHlI4=/62x34/filters:focal(500x168:510x178)/cloudfront-eu-central-1.images.arcpublishing.com/prisa/GNHCM22OMXZUKQPXEGA7HLFF3U.jpg",
    },
    "site": "elPais",
    "title": "La Audiencia de Madrid no ve delito de odio en el cartel de Vox contra los menores inmigrantes",
    "url": "/espana/madrid/2021-07-19/la-audiencia-de-madrid-no-ve-delito-de-odio-en-el-cartel-de-vox-contra-los-menores-inmigrantes.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Pedro Sánchez se prepara ya para 2024 ",
    "url": "/espana/2021-07-19/sanchez-se-prepara-ya-para-2024.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Podemos presiona al Gobierno para salvar el bloqueo judicial del PP",
    "url": "/espana/2021-07-19/podemos-presiona-al-gobierno-para-salvar-el-bloqueo-judicial-del-pp.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Europa, ante el dilema de la nueva ola de coronavirus",
    "url": "/sociedad/2021-07-19/europa-ante-el-dilema-de-la-nueva-ola-de-coronavirus.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La vida en Caracas bajo el poder de El Koki, el pandillero que desafía al chavismo",
    "url": "/internacional/2021-07-19/la-vida-en-caracas-bajo-el-poder-del-koki-el-pandillero-que-desafia-al-chavismo.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Un tribunal de Japón condena a dos estadounidenses por ayudar a Carlos Ghosn a fugarse del país",
    "url": "/economia/2021-07-19/un-tribunal-de-japon-condena-a-dos-estadounidenses-por-ayudar-a-carlos-ghosn-a-fugarse-del-pais.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "18 millones de niños viven, juegan y trabajan entre basura electrónica",
    "url": "/planeta-futuro/2021-07-19/viejas-computadoras-o-telefonos-18-millones-de-ninos-viven-juegan-y-trabajan-entre-basura-digital.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Brecha académica en la pandemia: las profesoras mejoraron sus currículos menos que ellos ",
    "url": "/educacion/2021-07-19/brecha-academica-en-la-pandemia-las-profesoras-engordaron-menos-sus-curriculos-que-ellos.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "León se repuebla con acento colombiano",
    "url": "/espana/2021-07-19/leon-se-repuebla-con-acento-colombiano.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Del ‘Donkey Kong’ al ‘Defender’: memorias del salón recreativo
",
    "url": "/cultura/2021-07-19/memorias-del-salon-recreativo.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La eterna juventud de los ‘abuelitos’ olímpicos",
    "url": "/deportes/juegos-olimpicos/2021-07-19/la-eterna-juventud-de-los-abuelitos-olimpicos.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Jackson Browne: “Las grandes estructuras económicas son estúpidas”",
    "url": "/cultura/2021-07-19/jackson-browne-las-grandes-estructuras-economicas-son-estupidas.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Ellas mandan en ‘La Unidad’: “Cuanto más maduras somos, más interesantes podemos ser en los personajes”",
    "url": "/television/2021-07-19/ellas-mandan-en-la-unidad-cuanto-mas-maduras-somos-mas-interesantes-podemos-ser-en-los-personajes.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Los ‘best sellers’ de marcas pequeñas españolas que se agotan sin parar",
    "url": "https://smoda.elpais.com/moda/los-best-sellers-de-marcas-pequenas-espanolas-que-se-agotan-sin-parar/100438487/image/100438488",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Rodrigo Cuevas: “Perder el arte del cortejo es el mayor signo de decadencia humana”",
    "url": "/icon/2021-07-19/rodrigo-cuevas-perder-el-arte-del-cortejo-es-el-mayor-signo-de-decadencia-humana.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Rebajas, ¿cómo compran ahora los españoles?",
    "url": "/economia/estar-donde-estes/2021-07-19/rebajas-como-compran-ahora-los-espanoles.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Estabilizar el mercado del crudo",
    "url": "/opinion/2021-07-19/estabilizar-el-mercado-del-crudo.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El poder",
    "url": "/opinion/2021-07-19/el-poder.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El Constitucional se autolesiona",
    "url": "/opinion/2021-07-19/el-constitucional-se-autolesiona.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Con Nelson Mandela ",
    "url": "/opinion/2021-07-19/con-nelson-mandela-hoy.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Tomarse la libertad en serio",
    "url": "/opinion/2021-07-19/tomarse-la-libertad-en-serio.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Las mil y una vidas de la catedral de Burgos",
    "url": "/cultura/2021-07-17/las-mil-y-una-vidas-de-la-catedral-de-burgos.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "De las llamas de Moria a las aguas del Rin: un nuevo comienzo para los Yussefi",
    "url": "/internacional/2021-07-18/de-las-llamas-de-moria-a-las-aguas-del-rin-un-nuevo-comienzo-para-los-yussefi.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Cuba rompe su silencio",
    "url": "/internacional/2021-07-18/el-alarido-que-ha-conmocionado-cuba.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Vida de Joe Ligon, encarcelado a los 15 años y puesto en libertad a los 83",
    "url": "/internacional/2021-07-18/vida-de-joe-ligon-encarcelado-a-los-15-anos-y-liberado-a-los-83.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Una guerra híbrida en el confín de Europa",
    "url": "/internacional/2021-07-19/una-guerra-hibrida-en-el-confin-de-europa.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Merkel introduce la agenda verde en la precampaña electoral alemana",
    "url": "/internacional/2021-07-19/merkel-introduce-la-agenda-verde-en-la-precampana-electoral-alemana.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Los mexicanos no ven necesidad de que España pida disculpas por la conquista
",
    "url": "/mexico/2021-07-18/los-mexicanos-creen-que-lopez-obrador-utiliza-la-conquista-para-hacer-politica-y-no-ven-necesidad-de-que-espana-pida-disculpas.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La izquierda y la derecha cambian de generación en Chile con miras a la presidencia en 2022  ",
    "url": "/internacional/2021-07-19/la-izquierda-y-la-derecha-cambian-de-generacion-en-chile-con-miras-a-la-presidencia-en-2022.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La última gran serie de ciencia ficción. Por Hugo Silva",
    "url": "https://tienesqueverlo.vodafone.es/raised-by-wolves-por-hugo-silva/?con=tienes_que_verlo:cp-tqv:cn-contenidos:dt-20210716:wn-marketing:st-prospecting:cl-no_cliente:sp-el_pais:pl-modulo_destacado:pr-television",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Otros asesinos de pueblo",
    "url": "https://tienesqueverlo.vodafone.es/si-te-gusto-mare-of-easttown/?con=tienes_que_verlo:cp-tqv:cn-contenidos:dt-20210716:wn-marketing:st-prospecting:cl-no_cliente:sp-el_pais:pl-modulo_destacado:pr-television",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Ester Capella: “Cataluña no suele ir a la conferencia de presidentes. Los vascos, tampoco”",
    "url": "/espana/catalunya/2021-07-19/ester-capella-cataluna-no-suele-ir-a-la-conferencia-de-presidentes-los-vascos-tampoco.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La gerente y el tesorero de Podemos niegan que una asesora de Irene Montero trabajara como niñera",
    "url": "/espana/2021-07-19/la-gerente-y-el-tesorero-de-podemos-niegan-que-una-asesora-de-irene-montero-trabajara-como-ninera.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Velando el cadáver equivocado",
    "url": "/espana/2021-07-19/velando-el-cadaver-equivocado.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La desigualdad que ceba a las bandas juveniles en Madrid ",
    "url": "/espana/madrid/2021-07-19/de-las-banlieu-a-las-bandas-juveniles-la-delincuencia-que-alimenta-la-desigualdad.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Los incendios amenazan a las zonas urbanizadas por la falta de lluvia",
    "url": "/espana/catalunya/2021-07-18/los-incendios-amenazan-a-las-zonas-urbanizadas-por-la-falta-de-lluvia.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La nova Catalunya de la covid",
    "url": "https://cat.elpais.com/cat/2021/07/16/actualidad/1626438702_843098.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El emprendedor podrá proteger su vivienda en los concursos",
    "url": "/economia/2021-07-19/el-emprendedor-podra-proteger-su-vivienda-en-los-concursos.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Las acciones de Ence se desploman ante la incertidumbre sobre su fábrica en Pontevedra",
    "url": "/economia/2021-07-19/las-acciones-de-ence-se-desploman-ante-la-incertidumbre-sobre-su-fabrica-en-pontevedra.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La Ley de la Cadena hace agua en la leche",
    "url": "/economia/2021-07-19/la-ley-de-la-cadena-hace-aguas-en-la-leche.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Toda la banca española se interesa por el 9,4% de Bizum que vende CaixaBank",
    "url": "https://cincodias.elpais.com/cincodias/2021/07/18/companias/1626622945_717550.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Un mensaje de pandemia a pandemia: “La política debe seguir a la ciencia, no politizarla”",
    "url": "/planeta-futuro/2021-07-19/un-mensaje-de-pandemia-a-pandemia-la-politica-debe-seguir-a-la-ciencia-no-politizarla.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Medicina personalizada, una nueva forma de asistencia sanitaria",
    "url": "/sociedad/2021-07-19/medicina-personalizada-una-nueva-forma-de-asistencia-sanitaria.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La sequía dispara la tensión en el campo andaluz",
    "url": "/clima-y-medio-ambiente/2021-07-19/la-sequia-dispara-la-tension-en-el-campo-andaluz.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Una dosis para muchos o dos dosis para pocos: el dilema que divide a América Latina en la lucha contra la pandemia ",
    "url": "/sociedad/2021-07-18/primera-dosis-para-muchos-o-dos-dosis-para-pocos-el-dilema-que-divide-a-america-latina-en-la-lucha-contra-la-pandemia.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Una misteriosa espuma blanca provoca el cierre de varias playas en México",
    "url": "/mexico/2021-07-18/una-misteriosa-espuma-blanca-provoca-el-cierre-de-varias-playas-en-veracruz.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Qué es WhatsApp Business y cómo utilizarlo mejor",
    "url": "/tecnologia/2021-07-19/que-es-whatsapp-business-y-como-utilizarlo-mejor.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "¿Mató el móvil al álbum de fotos familiar?",
    "url": "/tecnologia/2021-07-18/mato-el-movil-al-album-de-fotos-familiar.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Cuando salí de Cuba",
    "url": "/cultura/2021-07-19/cuando-sali-de-cuba.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Silvana Estrada, la canción triste de una ‘viejoven’ francamente feliz",
    "url": "/espana/madrid/2021-07-19/silvana-estrada-la-cancion-triste-de-una-viejoven-francamente-feliz.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Julia Ducournau: “Yo odio la provocación, quiero remover conciencias”",
    "url": "/cultura/2021-07-18/julia-ducournau-odio-la-provocacion.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Ariel Goldenberg, el hombre que no quiso tomar la Bastilla",
    "url": "/cultura/2021-07-19/ariel-goldenberg-el-hombre-que-no-quiso-tomar-la-bastilla.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "España muestra sus cartas y Estados Unidos enseña los dientes (76-83)",
    "url": "/deportes/juegos-olimpicos/2021-07-19/espana-muestra-sus-cartas-y-estados-unidos-ensena-los-dientes.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El escribano centenario de Messi ",
    "url": "/deportes/2021-07-19/el-escribano-centenario-de-messi.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Pogacar pone a una generación a sus pies",
    "url": "/deportes/2021-07-18/pogacar-consigue-su-segundo-tour-de-francia-consecutivo.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Morikawa gana el Open Británico y Jon Rahm recupera el número uno mundial",
    "url": "/deportes/2021-07-18/morikawa-gana-el-open-britanico-en-su-primera-participacion.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Mbappé sigue dando largas al PSG con su renovación",
    "url": "https://as.com/futbol/2021/07/18/internacional/1626617460_139409.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Un acuerdo millonario pone fin a la batalla judicial entre el creador de ‘The Walking Dead’ y AMC",
    "url": "/television/2021-07-19/un-acuerdo-millonario-pone-fin-a-la-batalla-judicial-entre-el-creador-de-the-walking-dead-y-amc.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Protegernos de los prejuicios de los padres",
    "url": "/television/2021-07-19/protegernos-de-los-prejuicios-de-los-padres.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Estella, capital navarra del Camino",
    "url": "https://elpais.com/elviajero/2021/07/15/actualidad/1626346693_639596.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Tus lecturas de verano empiezan por estos relatos para compartir y brindar con los amigos",
    "url": "https://smoda.elpais.com/moda/tus-lecturas-de-verano-empiezan-por-estos-relatos-para-compartir-y-brindar-con-los-amigos/",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Minimalista, tecnológico y respetuoso con el entorno: así es el oasis más futurista de Arabia Saudí",
    "url": "/icon-design/arquitectura/2021-07-19/minimalista-tecnologico-y-respetuoso-con-el-entorno-asi-es-el-oasis-mas-futurista-de-arabia-saudi.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El club de la  birra y la política  ",
    "url": "/eps/2021-07-19/el-club-de-la-birra-y-la-politica.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Tomar descafeinado sin morir de tristeza",
    "url": "https://elpais.com/elcomidista/2021/07/06/articulo/1625571048_926686.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La emotiva carta de despedida de Penélope Cruz a Pilar Bardem: “¡Siempre fuiste tan buena conmigo!”",
    "url": "/gente/2021-07-19/la-emotiva-carta-de-despedida-de-penelope-cruz-a-pilar-bardem-siempre-fuiste-tan-buena-conmigo.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Britney Spears renuncia a su carrera: “No voy a actuar con mi padre manejando lo que me pongo, digo, hago o pienso”",
    "url": "/gente/2021-07-18/britney-spears-alza-la-voz-no-voy-a-actuar-con-mi-padre-manejando-lo-que-me-pongo-digo-hago-o-pienso.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Familias vulnerables: así impactan las desigualdades en la salud y el rendimiento escolar de los niños ",
    "url": "/mamas-papas/2021-07-19/familias-vulnerables-asi-impactan-desigualdades-en-la-salud-y-el-rendimiento-escolar-de-los-ninos.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Sacrificios en cascada de Aronián",
    "url": "/elpais/2021/07/19/el_rincon_de_los_inmortales/1626685974_846886.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Ocho consejos sencillos para sobrevivir al volante en verano",
    "url": "https://motor.elpais.com/conducir/consejos-sencillos-coche-en-verano/",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Aston Martin Valhalla, un superdeportivo híbrido enchufable de casi 1.000 CV",
    "url": "https://motor.elpais.com/supercoches/el-aston-martin-valhalla-es-un-superdeportivo-hibrido-enchufable-de-950-cv/",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El sorprendente efecto del nuevo impuesto de matriculación",
    "url": "https://motor.elpais.com/actualidad/el-sorprendente-efecto-del-nuevo-impuesto-de-matriculacion/",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Sandra Sánchez, la karateca que fue marginada por estar al lado de su madre y que ha vuelto para luchar por el oro olímpico",
    "url": "/deportes/juegos-olimpicos/2021-07-14/videorreportaje-sandra-sanchez-la-karateca-que-fue-marginada-por-estar-al-lado-de-su-madre-y-que-ha-vuelto-para-luchar-por-el-oro-olimpico.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "‘Ikatza’",
    "url": "https://elpais.com/elpais/2021/07/12/album/1626081780_051910.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Los zuecos Crocs más vendidos para niños y adultos: en 35 colores y desde 20 euros en Amazon",
    "url": "/elpais/2020/05/26/escaparate/1590498254_064032.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Los mejores altavoces Bluetooth con radio",
    "url": "/escaparate/2021-07-19/los-mejores-altavoces-bluetooth-con-radio.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La crema mágica de La Roche Posay que los dermatólogos recomiendan para todo",
    "url": "/escaparate/2021-07-18/la-crema-magica-de-la-roche-posay-que-los-dermatologos-recomiendan-para-todo.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "PRISA Media ficha a Marisa Manzano como nueva directora general comercial",
    "url": "/comunicacion/2021-07-13/prisa-media-ficha-a-marisa-manzano-directora-comercial-de-facebook-en-espana.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "EL PAÍS estrena sección de pasatiempos",
    "url": "/comunicacion/el-pais-que-hacemos/2021-07-09/el-pais-estrena-seccion-de-pasatiempos.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Recibe cada semana los grandes temas de la actualidad: artículos y entrevistas para entender el mundo que nos rodea",
    "url": "https://plus.elpais.com/newsletters/lnp/1/398",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Taller sobre periodismo de datos",
    "url": "https://escuela.elpais.com/talleres/periodismo-de-datos-en-linea/",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Abierto el plazo de inscripción al Máster de Periodismo 2021-2023",
    "url": "https://escuela.elpais.com/inscripcion-al-master-de-periodismo-2021-2023/",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Llorar y luchar, las dos caras de la misma moneda en Yemen",
    "url": "/planeta-futuro/2021-07-18/llorar-y-luchar-las-dos-caras-de-la-misma-moneda-en-yemen.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "El diseñador que devolvió la alta costura a Balenciaga",
    "url": "/eps/2021-07-18/el-disenador-que-devolvio-la-alta-costura-a-balenciaga.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Tokio 2020: unos Juegos gafados para la economía japonesa",
    "url": "/economia/2021-07-10/tokio-2020-unos-juegos-gafados-para-la-economia-japonesa.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Kyushu, por el genuino sur de Japón",
    "url": "https://elpais.com/elviajero/2021/07/15/actualidad/1626358329_114064.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "82 libros para el verano",
    "url": "/babelia/2021-07-10/82-libros-de-verano.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "La meritocracia es una trampa ",
    "url": "/ideas/2021-07-18/la-meritocracia-es-una-trampa.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Guardar el dinero en casa: legal sí, pero no seguro",
    "url": "https://cincodias.elpais.com/cincodias/2021/07/16/midinero/1626418511_399898.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Ensalada de patata, pesto y morcilla patatera",
    "url": "https://elpais.com/elcomidista/2021/07/12/receta/1626084929_583136.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "17 objetos rebajados de Ikea que no deberías dejar escapar (Billy incluida)",
    "url": "/elpais/2021/07/05/album/1625483230_667213.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "20 famosos veganos explican por qué dejaron de comer carne",
    "url": "https://smoda.elpais.com/celebrities/vips/famosos-veganos-motivos/100438119/image/100438138",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Isabel Rodríguez pide a la ciudadanía \\"un último esfuerzo\\" de \\"prudencia individual\\" frente a la covid",
    "url": "https://cadenaser.com/programa/2021/07/19/hoy_por_hoy/1626665963_114938.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "A Egea le llueven las críticas por lo que dijo de Sánchez: \\"Debe creer que somos tontos\\"",
    "url": "https://www.huffingtonpost.es/entry/a-egea-le-llueven-las-criticas-por-lo-que-dijo-de-sanchez-debe-creer-que-somos-tontos_es_60f50e05e4b0a771e805da19",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Verstappen estalla contra Hamilton desde el hospital",
    "url": "https://as.com/motor/2021/07/18/formula_1/1626616611_869143.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Juegos gratis de julio en PS Plus, Xbox Gold, Prime Gaming y Stadia Pro",
    "url": "https://as.com/meristation/2021/07/05/noticias/1625474949_076371.html",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Descubre la suscripción que mejor se adapta a ti",
    "url": "http://suscripciones.elpais.com/?id_externo_promo=suscrip-pie-home",
  },
  Object {
    "images": Object {
      "default": undefined,
    },
    "site": "elPais",
    "title": "Disfruta del Festival de Teatro Clásico de Mérida",
    "url": "https://elpais.com/tematicos/2021/06/15/elpaismas/1623755625_211453.html",
  },
]
`;
